command/config/util.go

```diff
@@ -0,0 +1,40 @@
+package config
+
+import (
+	"bytes"
+	"encoding/json"
+	"io"
+	"os"
+
+	"github.com/ghodss/yaml"
+)
+
+func ProcessPoolFile(rawFile string) (*PoolFile, error) {
+	rawPool, err := os.ReadFile(rawFile)
+	if err != nil {
+		return nil, err
+	}
+	data := io.NopCloser(
+		bytes.NewBuffer(rawPool),
+	)
+	inst, err := Parse(data)
+	if err != nil {
+		return nil, err
+	}
```

Comment:
There's no reason to read the whole file just to put it into a buffer and create a reader from the buffer. A file already is a `io.Reader`.

```suggestion
	f, err := os.Open(rawFile)
	if err != nil {
		return nil, err
	}
	defer f.Close()
	
	inst, err := Parse(f)
	if err != nil {
		return nil, err
	}
```

Reply:
updated